ProfileTOS.md

```diff
@@ -26,25 +26,30 @@ Because we can and want you to know what we save and do with your information.
   "songsPlayed": <the amount of songs you played>,
   "isDonator": <if you are a donator>,
   "amountDonated": <how much you donated so far>,
-  "customRolename": "<your custom rolename, donator feature>",
-  "customRoleColor": "<your custom rolecolor, donator feature>",
-  "customWebBackground": "<your custombanner url, for your web and client profile>",
+  "customRolename": "<your custom role name, donator feature>",
+  "customRoleColor": "<your custom role color, donator feature>",
+  "customWebBackground": "<your custom banner URL, for your web and client profile>",
   "profilePicture": "<if you have uploaded a profile picture, DEPRECATED>"
 }
 ```
-Your activity (XP and songs played) updates whenever you're listening to your songs, while beeing logged in.
+Your activity, which includes: XP, songs played, updates on whether you're listening to songs while logged in  
 The above information is publicly visible by visiting our [stats page](https://stats.founntain.dev) or by viewing profiles inside the client.
 
-Thats all the data we store of you in our *database*, when you delete your profile the above object of you is gone and erased.  
-Your profile picture will also be deleted when you delete your profile, obviously.
+That's all the data we store of you in our *database*, when you delete your profile the above object of you is gone and erased.  
+Your profile picture will also be deleted when you delete your profile.
 
 ### ☝️ Some rules
-Don't post anything NSFW, hate-/harmful in your description or somewhere else in the client. This also includes your profile picture. *(sorry about that, but no anime tiddies as profile picture)*  
+Don't post anything NSFW, hate-/harmful in your description or somewhere else in the client. This also includes your profile picture. *(sorry about that, but no anime titties as profile picture)*  
 
-If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create on if you intend to use the player and its user feature aswell and as intended.  
-You don't need a profile if you don't use the user feature to collect XP and statistics! If you profile was deleted, you will still be able to create a new profile if you think you use it as intended.
+If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create an account if you intend to use the player and its user feature as intended.  
+You don't need a profile if you do not want to use the user features, such as collecting XP and statistics! If your profile was deleted, you will still be able to create a new profile if you believe you will use it as intended.
 
-⚠️ If for whatever reason you do so, we have the right to clear your description, remove your profile picture / banner or send your profile to the *sun (deleting it)*.
+## 📫 Contact us
```

Comment:
Maybe swap the contact use with the warning from beloe. so warning first anf then contact

Reply:
done